engine/src/main/java/org/terasology/config/PlayerConfig.java

```diff
@@ -25,66 +25,126 @@
+    private Color color = defaultPlayerColor(); // Sets player's default color
 
-    private Float height = DEFAULT_PLAYER_HEIGHT;
+    private Float height = DEFAULT_PLAYER_HEIGHT; // Sets player's default height
 
-    private Float eyeHeight = DEFAULT_PLAYER_EYE_HEIGHT;
+    private Float eyeHeight = DEFAULT_PLAYER_EYE_HEIGHT;  // Sets player's default eye height
 
-    private boolean hasEnteredUsername;
+    private boolean hasEnteredUsername; // Property regarding whether or not player has entered username
 
+    /**
+     * Gets the player's name
+     *
+     * @return   A String representing the player's name
+     */
     public String getName() {
         return name;
     }
-
+    
+    /**
+     * Sets the player's name
+     *
+     * @param name  A String representing the player's desired name
+     */    
     public void setName(String name) {
         this.name = name;
     }
-
+    
+    /**
+     * Gets the player's color
+     *
+     * @return   A Color object representing the player's color
+     */
     public Color getColor() {
         return color;
     }
-
+    
+    /**
+     * Sets the player's color
+     *
+     * @param color  A Color object representing the player's desired color
+     */
     public void setColor(Color color) {
         this.color = color;
     }
 
+    /**
+     * Gets the player's height
+     *
+     * @return   A float representing the player's height
+     */
     public Float getHeight() {
         return height;
     }
-
+    
+    /**
+     * Sets the player's height
+     *
+     * @param height  A float representing the player's desired height
+     */
     public void setHeight(Float height) {
         this.height = height;
     }
 
+    /**
+     * Gets the player's eye height
+     *
+     * @return  A float representing the player's eye height
+     */
     public Float getEyeHeight() {
         return eyeHeight;
     }
-
+    
+    /**
+     * Sets the player's eye height
+     *
+     * @param eyeHeight  A float representing the player's desired eye height
+     */
     public void setEyeHeight(Float eyeHeight) {
         if (eyeHeight < this.height) {
             this.eyeHeight = eyeHeight;
         }
     }
 
+    /**
+     * Checks to see if the player has entered a username
+     *
+     * @return  A boolean returning true if the player has entered a username
+     */    
     public boolean hasEnteredUsername() {
         return hasEnteredUsername;
     }
 
+    /**
+     * Configures the property of whether or not the player has entered a username
+     *
+     * @param entered  A boolean representing the desired state of the hasEnteredUsername property
+     */ 
     public void setHasEnteredUsername(boolean entered) {
         this.hasEnteredUsername = entered;
     }
 
+    /**
+     * Returns a String representing the player's default name
+     *
+     * @return A String representing the player's default name
+     */ 
```

Comment:
In this case, i would like to see a different javadoc. 
I know that the method returns the players default name by reading the method name. However i don't know, that the default name will be "Player", followed by a 5-digit random number.